po/ja.po

```diff
@@ -7033,33 +7200,50 @@ msgstr "選択したオペレーティングシステムには、必要なメモ
 msgid ""
 "The selected Operating System has minimum storage size requirement of $0 $1"
 msgstr ""
+"選択したオペレーティングシステムには、最小ストレージザイズ $0 $1 が必要です。"
 
 #: src/ws/login.js:673
 msgid ""
 "The server refused to authenticate '$0' using password authentication, and "
 "no other supported authentication methods are available."
-msgstr "サーバーはパスワード認証を使用した '$0' の認証を拒否しました。サポートされた他の認証方法は利用できません。"
+msgstr ""
+"サーバーはパスワード認証を使用した '$0' の認証を拒否しました。サポートされた"
+"他の認証方法は利用できません。"
 
 #: src/base1/cockpit.js:4034
 msgid "The server refused to authenticate using any supported methods."
 msgstr "サーバーはサポートされた方法を使用した認証を拒否しました。"
 
-#: pkg/systemd/hwinfo.jsx:71
+#: pkg/systemd/hwinfo.jsx:55
 msgid "The user $0 is not permitted to change cpu security mitigations"
 msgstr "ユーザー $0 は、cpu セキュリティー緩和の変更を許可されていません"
 
-#: pkg/systemd/init.js:819
+#: pkg/systemd/overview-cards/configurationCard.jsx:355
+msgid "The user $0 is not permitted to change the system time"
+msgstr ""
+
+#: pkg/systemd/overview-cards/configurationCard.jsx:335
+msgid "The user $0 is not permitted to modify hostnames"
+msgstr ""
+
+#: pkg/systemd/overview-cards/configurationCard.jsx:361
+#, fuzzy
+#| msgid "The user <b>$0</b> is not permitted to modify realms"
+msgid "The user $0 is not permitted to modify realms"
+msgstr "ユーザー <b>$0</b> はレルムを変更することを許可されていません"
```

Comment:
This looks wrong as well.

Reply:
Indeed, someone unfuzzed without applying the difference. We just dropped the `<b>` though, we didn't change the string. I fixed it in weblate.